src/helm/proxy/clients/openai_client.py

```diff
@@ -19,6 +21,7 @@
 
 class OpenAIClient(Client):
     END_OF_TEXT: str = "<|endoftext|>"
+    MODELS_USING_TIKTOKEN: List[str] = ["gpt-3.5-turbo-0301", "gpt-4-32k-0314"]
```

Comment:
Pretty sure this isn't used anywhere now.

Reply:
Yep. Done.